generators/kubernetes/templates/_service.yml

```diff
@@ -20,11 +20,19 @@ apiVersion: v1
 kind: Service
```

Comment:
Do you want to move this object to _deployment.yml. If you look at other templates kafka, mysql, registry etc both the deployment and service objects in the same file. To make it consistent.